constants.py

```diff
@@ -15,8 +15,8 @@
 
 import os
 
+from aws_cdk import Environment
```

Comment:
I'd prefer to be explicit about `Environment`'s module, since this is quite a generic word:
```suggestion
import aws_cdk as cdk
```